docs/pages/tutorials/kafka-tutorial.md

```diff
@@ -49,6 +49,7 @@ configs:
 ```
 
 9. Save and exit the `svc.yml` file. It should look similar to the following:
+
```

Comment:
unintentional changes in this PR?

Reply:
Not entirely unintentional, just sneaking it in to fix a formatting problem: http://mesosphere.github.io/dcos-commons/tutorials/kafka-tutorial.html